src/graphics/program-lib/chunks/aoMap.frag

```diff
@@ -0,0 +1,18 @@
+#ifdef MAPTEXTURE
```

Comment:
This define applies to all maps? How come we don't need one for each map type?

Reply:
No. Shader generator #undefs previous defines before every map: https://github.com/playcanvas/engine/pull/1080/files#diff-0df44c31b04313738bd9cf0de37958bbR173